.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: php
+
+sudo: true
+
+addons:
+  firefox: "47.0.1"
+  postgresql: "9.4"
+
+services:
+  - mysql
+  - postgresql
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.npm
+
+php:
+ - 7.0
+ - 7.1
+ - 7.2
+
+env:
+ global:
+  - MOODLE_BRANCH=MOODLE_35_STABLE
+  - MOODLE_BRANCH=MOODLE_38_STABLE
+ matrix:
+  - DB=pgsql
+  - DB=mysqli
+
+before_install:
+  - phpenv config-rm xdebug.ini
+  - nvm install 14.2.0
+  - nvm use 14.2.0
+  - cd ../..
+  - composer create-project -n --no-dev --prefer-dist blackboard-open-source/moodle-plugin-ci ci ^2
```

Comment:
can you please change this to the new moodlehq template at: 
https://github.com/moodlehq/moodle-plugin-ci/blob/master/.travis.dist.yml

(MoodleHQ recently took over the travis config plugin.)